NumPy/SciPy nightly job failing in CI

The NumPy/SciPy nightly job is failing in all CI runs:
https://github.com/sympy/sympy/actions/runs/10110311336/job/27960035596

The failed job shows
```
sympy/stats/sampling/sample_scipy.py:112: in _pmf
    return handmade_pmf(x)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Dummy_53199 = array([ 1,  2,  3,  4,  5,  6,  7,  8,  9, 10, 11, 12, 13])

    def _lambdifygenerated(Dummy_53199):
>       return (2/3)*3**(1 - Dummy_53199)
E       ValueError: Integers to negative integer powers are not allowed.

<lambdifygenerated-1314>:2: ValueError
                                DO *NOT* COMMIT!                                
=========================== short test summary info ============================
FAILED sympy/stats/sampling/tests/test_sample_discrete_rv.py::test_sample_scipy - ValueError: Integers to negative integer powers are not allowed.
```

I believe this is due to some recent change in SciPy main so I have opened a SciPy issue: https://github.com/scipy/scipy/issues/21272

--- a/sympy/stats/sampling/tests/test_sample_discrete_rv.py
+++ b/sympy/stats/sampling/tests/test_sample_discrete_rv.py
@@ -1,7 +1,17 @@
 from sympy.core.singleton import S
-#from sympy.core.symbol import Symbol
+from sympy.core.symbol import Symbol
 from sympy.external import import_module
-from sympy.stats import Geometric, Poisson, Zeta, sample, Skellam, Logarithmic, NegativeBinomial, YuleSimon
+from sympy.stats import (
+    Geometric,
+    Poisson,
+    Zeta,
+    sample,
+    Skellam,
+    Logarithmic,
+    NegativeBinomial,
+    YuleSimon,
+    DiscreteRV,
+)
 from sympy.testing.pytest import skip, raises, slow
 
 
@@ -27,13 +37,11 @@ def test_sample_numpy():
 
 
 def test_sample_scipy():
-    #p = S(2)/3
-    #x = Symbol('x', integer=True, positive=True)
-    #pdf = p*(1 - p)**(x - 1) # pdf of Geometric Distribution
+    p = S(2)/3
+    x = Symbol('x', integer=True, positive=True)
+    pdf = p*(1 - p)**(x - 1) # pdf of Geometric Distribution
     distribs_scipy = [
-        # This one fails:
-        #    https://github.com/sympy/sympy/issues/26862
-        # DiscreteRV(x, pdf, set=S.Naturals),
+        DiscreteRV(x, pdf, set=S.Naturals),
         Geometric('G', 0.5),
         Logarithmic('L', 0.5),
         NegativeBinomial('N', 5, 0.4),

